routes/userRouteNotes.js

```diff
@@ -0,0 +1,57 @@
+router.get('/widgets/:widgetId', ( request, response, next ) => {
```

Comment:
Please rename `:widgetId` to `:id` - it's implied that this is a widget ID by the route